dev/com.ibm.ws.jbatch.open_fat/fat/src/batch/fat/junit/BasicJDBCPersistenceTestCheckpoint.java

```diff
@@ -0,0 +1,57 @@
+package batch.fat.junit;
+
+import org.jboss.shrinkwrap.api.Archive;
+
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import batch.fat.util.BatchFATHelper;
+
+import com.ibm.ws.jbatch.test.BatchAppUtils;
+import com.ibm.ws.jbatch.test.FatUtils;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.custom.junit.runner.FATRunner;
+import com.ibm.ws.jbatch.test.FatUtils;
+import componenttest.annotation.CheckpointTest;
+import io.openliberty.checkpoint.spi.CheckpointPhase;
+
+@RunWith(FATRunner.class)
```

Comment:
Missing `@CheckpontTests`.

This test case isn't configured into the `FATSuite`.  Did you intend for it to be?

Reply:
Yes, this should be part of the FATSuite. Thank you Tom!

Reply:
This class needs to be added to the `FATSuite`

Reply:
Yes, I accidentally added the  "@CheckpointTest" within the class, correcting this and the other review as well. Thank you Tom!

Reply:
Both are needed.  The `@CheckpointTest` annotation is needed on this class.  And this class needs to be included in the list of classes in the `FATSuite`.

Reply:
Sounds good, updated the FATSuite as well